### ISSUE TITLE
dbt cluster configs

### ISSUE THREAD

**Issue by jnoynaert** [MaterializeInc/materialize#15152]
### Feature request

To effectively use the dbt adapter with Materialize Cloud, cluster configs should be exposed as a model-level or project config.

Ideally we should be able to apply this via model `config` blocks in the model definition file, or in `dbt_project.yml` as below:

```yml
models:
  my_namespace:
    use_case_one:
      +schema: one
      +cluster: big_pappa
      +materialized: materializedview
    use_case_two:
      +schema: two
      +cluster: cluster_b
      +materialized: materializedview
```

**Comment by jnoynaert** [comment]
Just kidding, looks like this is already available in 1.2.0

**Comment by morsapaes** [comment]
Yeah! It's also finally documented, @jnoynaert: https://materialize.com/docs/integrations/dbt/#configuration
